_posts/2021-06-30-community-update

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'June Community Update'
+date: 2021-06-30 00:00:00 
+author: octagon
+image: '/images/6.jpg'
+published: false
+---
+
+Dear Frentacles,
+
+It's been a while since our last update but we're here and we're strong.
+
+Throughout this month, a few of our Octagon members had some real-life events that needed to be taken care of which delayed our progress and community updates. We have also identified all issues that made it difficult for us to move forward. These included access to passwords, documents, and team wallets that were shared by Doc with select individuals, but not known among the full team. Many of these issues have been resolved and moving forward we will continue to improve communication internally and with the community to ensure the project has no single point of failure. You can expect to see regular updates and the creation of documentation and clear guidelines for all processes (open-source everything).
+
+While working through issues, the Octagon attempted to contact Doc by email multiple times for input on a few items (e.g. update salary streams). Unfortunately, we haven't received a reply. If he is reading this, please check your email. Regardless of whether we receive a response, we will continue to move forward.
+
+Development Updates
+-------------------
+
+In May we published our 'Development and Marketing Priorities Q2/Q3' blog post. In the update we identified all the UI/UX and product improvements that need to be addressed to make OctoFi a platform with a clear purpose and path toward user adoption. If you want to take a closer look at the development side of things, you can view [todo.octo.fi](http://todo.octo.fi), [milestones.octo.fi](http://milestones.octo.fi), or [priority.octo.fi](http://priority.octo.fi).
```

Comment:
Change `Development and Marketing Priorities Q2/Q3` to a link to the original article and remove the quotes around it.